### PR TITLE
Added a test to verify ReflectionTypeLoadException

### PR DESCRIPTION
**Pull request by ajaybhargavb**
Follow up to https://github.com/aspnet/Diagnostics/pull/317

@pranavkm

### CODE REVIEW

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs [hunk 1]
```diff
@@ -115,5 +115,23 @@ public ErrorPageTests(MvcTestFixture<ErrorPageMiddlewareWebSite.Startup> fixture
...
+            var expectedMediaType = MediaTypeHeaderValue.Parse("text/html; charset=utf-8");
+
+            // Act
+            var response = await Client.GetAsync("http://localhost/LoaderException");
+
+            // Assert
+            Assert.Equal(HttpStatusCode.InternalServerError, response.StatusCode);
+            Assert.Equal(expectedMediaType, response.Content.Headers.ContentType);
+            var content = await response.Content.ReadAsStringAsync();
+            Assert.Contains("LoaderExceptions:", content);
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs]: Might be better to add a message to the loader exception you added and verify that. Also, did we not change the text for this string to say "Loader Exceptions" (with the space?)

**Comment by pranavkm** [comment]
:watch:

File: test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs (hunk 1 above)
Comment (ajaybhargavb) [inline test/Microsoft.AspNetCore.Mvc.FunctionalTests/ErrorPageTests.cs]: Will do. I changed that in Hosting but not in Diagnostics. I'll update.

**Comment by ajaybhargavb** [comment]
:up: :date:

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**